presto-docs/src/main/sphinx/release/release-0.225.rst

```diff
@@ -0,0 +1,53 @@
+=============
+Release 0.225
+=============
+
+General Changes
+_______________
+* Fix the issue that fragment's output size was under-recorded for cases where input data is dictionary or run-length encoded, or the rows are replicated. (:issue:`11770`)
+* Fix leak in operator peak memory computations. (:issue:`13210`)
+* Allow overriding session zone sent by clients. (:issue:`13140`)
+* Improve memory tracking for spatial query indexing and avoid worker node crash due to out of memory. (:pr:`13251`)
+* Throw an exception when a spatial partition is made with 0 rows.
+* Add Flatbush RTree to replace JTS STR RTree for spatial query.
```

Comment:
I'm not sure we need a release note for this.  What's the user facing implication?

Reply:
Agree, this is implementation detail.